doc/help/kdb.md

```diff
@@ -76,7 +76,7 @@ For example, to permanently change verbosity one can use:
 
 Profiles allow users to change many/all configuration options of a tool
 at once. It influences from where the KDB entries are read.
-For example if you use:
+For example if you use:  
```

Comment:
Did you add the whitespace here and in line 91 intentionally?

Reply:
Yes, unfortunately Markdown requires such whitespaces to indicate a line break (but not a paragraph break). Without these two spaces the resulting man page is ugly. (Example would not start in a new line.)

I also dislike it but it seems to be the way how it is done in Markdown.

Reply:
Can we please (please please 🙏) replace these two trailing spaces with the HTML tag `<br />`. The tag should have exactly the same effect as the two trailing spaces, but makes your intention much clearer to someone who looks at the plain text file.

Reply:
If all of our 4 markdown renderer (website, ronn, GitHub, discount in qt-gui) support the `<br>` I also prefer to use that. Then we can systematically replace all occurrences of two spaces.

Reply:
> If all of our 4 markdown renderer (website, ronn, GitHub, discount in qt-gui) support the `<br>` I also prefer to use that.

I tried GitHub’s Markdown renderer, `discount` (`qt-gui`) and `ronn`. The tag `<br>` works in all of these applications. I assume the Website supports the `<br>` tag too.

I already replaced every occurrence of two or more spaces at the end of a line with the `<br>` tag in my local copy of the repo. The output of `ronn` looks a little bit different in some places, while in other places `ronn` produces exactly the same output as before. I will open a pull request containing the changes later.